examples/hello_world/src/lib.rs

```diff
@@ -61,3 +57,14 @@ impl oak::Node for Node {
         };
     }
 }
+
+// TODO: Generate parts of this code via a macro or code generation (e.g. a protoc plugin).
```

Comment:
Isn't this the only part that cannot be generated, i.e. it's the user provided logic?

Reply:
I had this thought yesterday that it would be nice to have a "proto" generator and an "wizard". You could manually create the proto and cargo files and start writing your app or you can use the "wizard" and say something like - new app with this proto file and we will generate everything, including the lib.rs that will be something like 
```
impl HelloWorldNode for Node {
   fn say_hello(&self, _: &HelloRequest) -> HelloResponse {
        let mut res = HelloResponse::new();
        // TODO: write your logic here
        res
    }
}
```

Reply:
I would prefer that we generate as much as possible statically from the service definition, and then provide a (non-generated) library of helpers (or "wizards") for common operations. Customising the behaviour of the generator will likely be more trouble than it's worth -- though feel free to try it out if you have any ideas about it already! Generating lib.rs seems unlikely -- where would the non-generated code go?